docs/package-sign.md

```diff
@@ -84,202 +84,69 @@ Follow these steps to get the packaging and signing tool and set up the virtual
-
-jar signed.
-
-Warning:
-
-No -tsa or -tsacert is provided and this jar is not timestamped. Without a timestamp, users may not be able to validate this jar after the signer certificate's expiration date (2020-10-06) or after any future revocation date.
-
-taco was signed as mysql_odbc_sample.taco at E:\connector-plugin-sdk\connector-packager\packaged-connector
-
 ```
 
 #### Example 2: .taco file is generated into user supplied location and log file is generated into user supplied location 
 
 ```
-(.venv) E:\connector-plugin-sdk\connector-packager>python -m connector_packager.package ..\samples\plugins\mysql_odbc -a test1year -ks test1yearkeystore.jks --dest e:\temp -l e:\temp
+(.venv) E:\connector-plugin-sdk\connector-packager>python -m connector_packager.package ..\samples\plugins\mysql_odbc  --dest e:\temp -l e:\temp
 
 Validation succeeded.
 
 mysql_odbc_sample.taco was created in e:\temp
 
-Enter keystore password:
-
-Enter password for alias test1year:(RETURN if same as keystore password)
-
-jar signed.
-
-Warning:
-
-No -tsa or -tsacert is provided and this jar is not timestamped. Without a timestamp, users may not be able to validate this jar after the signer certificate's expiration date (2020-10-06) or after any future revocation date.
-
 ```
 
+## Signing your packaged connector using jarsigner
 
-### Verify the connector with signature
-
-**Note:** You can use the package tool to package and sign the connector, and use the connector in Tableau Desktop 2019.x. Signature verification is supported starting in Tableau Desktop 2019.4 on Windows.  
-
-You can install and test your connector one of two ways:
-
--   Put your .taco file in the standard location, My Tableau Repository/Connectors, then launch Tableau Desktop. 
-
--   Put your .taco file anywhere you want. Launch Tableau Desktop with the packaged and signed connector with the option -DConnectPluginsPath . For example:
-
-    ```
-    "C:\Program Files\Tableau\Tableau 2019.4\bin\tableau.exe" -DConnectPluginsPath="d:\connector-plugin-sdk\connector-packager\packaged-connector" 
-    ```
-
-In both cases, when you launch Tableau Desktop, your connector is listed on the **Connect** pane, under **To a Server** > **More**.
+At this point, your connector has been packaged into a single `.taco` file. However, will not be loaded automatically into Tableau unless you sign the file, or disable signature verification.
 
-![]({{ site.baseurl }}/assets/connect-pane.png) 
- 
-
-To connect, select the name of your connector, enter the information in the Connection dialog, and then click **Sign In**.
+### Why we require connectors to be signed
 
-![]({{ site.baseurl }}/assets/data-source-page.png)
+Connectors are sensitive parts of the Tableau code. They handle with database authentication and communicate directly with your driver. We require that connector authors have gone through the steps to obtain a trusted certificate so that our customers can be confidant their data is safe.
```

Comment:
Do we want to state the points  of authenticity (which mean they taco developers are actually who they claim to be) and integrity (which basically mean that the data is not tampered with since it was created)

Reply:
Good idea, added it.